GitUI/GitUICommands.cs

```diff
@@ -1690,6 +1694,37 @@ public bool StartFileEditorDialog(string? filename, bool showWarning = false)
             return formEditor.ShowDialog() != DialogResult.Cancel;
         }
 
+        public void StartLocalFileEditorDialog(string? filename)
```

Comment:
GitUICommands doesn't look like the best place for this method, there's nothing git repo specific here.
`StartFileEditorDialog` shouldn't be here either, it's just a historical techdebt that needs to be paid...

Reply:
Okay. But paid to where? If there's another file full of commands like this, it's not in an obvious or easy to find place.

Reply:
My bad, I was thinking about GitModule.